File: example-java/src/main/java/com/pickth/imageslider/example/java/MainActivity.java
package com.pickth.imageslider.example.java;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.pickth.imageslider.listener.OnImageTouchListener;
import com.pickth.imageslider.view.ImageSlider;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    ArrayList items = new ArrayList<Integer>();
    items.add(R.drawable.a);
    items.add(R.drawable.b);
    items.add(R.drawable.c);

    ImageSlider is = findViewById(R.id.is_main);
    is.setOnImageTouchListener(new OnImageTouchListener() {
      @Override
      public void onClickListener(int position) {
        // do something
      }

      @Override
      public void onLongClickListener(int position) {
        // do something
      }
    });
    is.addItems(items);
    is.addItem(R.drawable.a);
  }
}

File: example/src/main/kotlin/com/pickth/imageslider/example/MainActivity.kt
package com.pickth.imageslider.example

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import com.pickth.imageslider.listener.OnImageTouchListener
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var items = ArrayList<Int>().apply {
            add(R.drawable.habit)
            add(R.drawable.movie_image)
            add(R.drawable.movie_image)
            add(R.drawable.movie_image)
        }

        is_main.setOnImageTouchListener(object: OnImageTouchListener {
            override fun onClickListener(position: Int) {
                // do something
            }

            override fun onLongClickListener(position: Int) {
                // do something
            }
        })

        is_main.addItems(items)
    }
}
